week-2/Homework/mandatory/2-function-creation.js

```diff
@@ -26,6 +46,10 @@ The function must:
 */
 
 function remove(arr, index) {
+  let newIndex=arr.slice(0, index) // removes the item at the specified index
+  let newIndex2=arr.slice(index+1)
+  let newArray = newIndex.concat(newIndex2)
+  return newArray; 
   return; // complete this statement
```

Comment:
A forgotten extra return.

Other than those few minor issues, this is great! Good job! 🎉